src/AdScreenerTool.js

```diff
@@ -240,13 +245,52 @@ const AdScreener = (params) => {
   	setSelectedTopicOrFullTextSearchTab(k);
   	if(k=='topics') {
   		setFullTextSearchQuery(null);
+		setPageId(null);
   	}
+	else if(k=='advertiser') {
+		setFullTextSearchQuery(null);
+		setTopic({ selectedOption: "" });
+		setTopicParam(undefined);
+	}
   	else {
-		setTopic({ selectedOption: ""});
+		setTopic({ selectedOption: "" });
```

Comment:
please revert this line since there is no functional change. This way the line by line change view is easier to read when doing code archeology